places/prague.md

```diff
@@ -0,0 +1,4 @@
+# Prague, CZ
+
+- Located in the heart of Europe.
+- You can eat anywhere :) It is relatively cheap.
```

Comment:
I'm going to Prague in April. I'll need some more specific recommendations, @jiricermak.